src/runtime_src/core/tools/common/XBHelpMenus.cpp

```diff
@@ -221,85 +209,83 @@ XBUtilities::produce_reports( xrt_core::device_collection devices,
 
   if(dev_report()) {
     // -- Process reports that work on a device
-    boost::property_tree::ptree ptDevices;
-    int dev_idx = 0;
-    for (const auto & device : devices) {
-      boost::property_tree::ptree ptDevice;
-      auto bdf = xrt_core::device_query<xrt_core::query::pcie_bdf>(device);
-      ptDevice.put("interface_type", "pcie");
-      ptDevice.put("device_id", xrt_core::query::pcie_bdf::to_string(bdf));
-
-      bool is_mfg = false;
-      try {
-        is_mfg = xrt_core::device_query<xrt_core::query::is_mfg>(device);
-      } 
-      catch (const xrt_core::query::exception&) {
-        is_mfg = false;
-      }
+    boost::property_tree::ptree ptDevice;
+    auto bdf = xrt_core::device_query<xrt_core::query::pcie_bdf>(device);
+    ptDevice.put("interface_type", "pcie");
+    ptDevice.put("device_id", xrt_core::query::pcie_bdf::to_string(bdf));
 
-      //if factory mode
-      std::string platform;
-      try {
-        if (is_mfg) {
-          platform = "xilinx_" + xrt_core::device_query<xrt_core::query::board_name>(device) + "_GOLDEN";
-        }
-        else {
-          platform = xrt_core::device_query<xrt_core::query::rom_vbnv>(device);
-        }
-      } 
-      catch(const xrt_core::query::exception&) {
-        // proceed even if the platform name is not available
-        platform = "<not defined>";
-      }
-      const std::string dev_desc = (boost::format("%d/%d [%s] : %s\n") % ++dev_idx % devices.size() % ptDevice.get<std::string>("device_id") % platform).str();
-      consoleStream << std::endl;
-      consoleStream << std::string(dev_desc.length(), '-') << std::endl;
-      consoleStream << dev_desc;
-      consoleStream << std::string(dev_desc.length(), '-') << std::endl;
-
-      const auto is_ready = xrt_core::device_query<xrt_core::query::is_ready>(device);
-      bool is_recovery = false;
-      try {
-        is_recovery = xrt_core::device_query<xrt_core::query::is_recovery>(device);
+    bool is_mfg = false;
+    try {
+      is_mfg = xrt_core::device_query<xrt_core::query::is_mfg>(device);
+    } 
+    catch (const xrt_core::query::exception&) {
+      is_mfg = false;
+    }
+
+    //if factory mode
+    std::string platform;
+    try {
+      if (is_mfg) {
+        platform = "xilinx_" + xrt_core::device_query<xrt_core::query::board_name>(device) + "_GOLDEN";
       }
-      catch(const xrt_core::query::exception&) { 
-        is_recovery = false;
+      else {
+        platform = xrt_core::device_query<xrt_core::query::rom_vbnv>(device);
       }
+    } 
+    catch (const xrt_core::query::exception&) {
+      // proceed even if the platform name is not available
+      platform = "<not defined>";
+    }
+    const std::string dev_desc = (boost::format("[%s] : %s\n") % ptDevice.get<std::string>("device_id") % platform).str();
+    consoleStream << std::endl;
+    consoleStream << std::string(dev_desc.length(), '-') << std::endl;
```

Comment:
Add a comment to let the user that this code is bounding the device description with minus '-'